extension.bundle.ts

```diff
@@ -15,6 +15,7 @@
 // The tests should import '../extension.bundle'. At design-time they live in tests/ and so will pick up this file (extension.bundle.ts).
 // At runtime the tests live in dist/tests and will therefore pick up the main webpack bundle at dist/extension.bundle.js.
 export * from '@microsoft/vscode-azext-utils';
+export * from '@microsoft/vscode-azext-azureutils';
```

Comment:
I'm pretty sure that you have to make sure that this export is below `azext-utils`. The reason being, `azureutils` relies on the UI extension variables being registered, so if you try to export this first, it breaks.

This is annoying because the `source.organizeImports` automatically brings it up to the top if you save with formatting on... 🤦

Reply:
Since I'm only importing `LocationListStep` now, the export being on top doesn't break anything, it seems.

Reply:
Whoa, I get it, but that's super weird.  Is this the first time we've run into this issue in `extension.bundle.ts` in an extension??